P1-Infrastructure/Identity/ApplicationUser.cs

```diff
@@ -1,11 +1,13 @@
 using Microsoft.AspNetCore.Identity;
+using P1_Application.Boundaries;
 using P1_Core.Entities;
 using P1_Core.Interfaces;
 
 namespace P1_Infrastructure.Identity
 {
+    //TODO change the name of this to something less confusing for the layered architecture
     public class ApplicationUser : IdentityUser
     {
-        public DiscordUser DiscordUser { get; set; }
+        public DiscordUser DiscordUser { get; set; } = null!;
```

Comment:
I am guessing the `null!` is a placeholder for the DiscordUser right now as we figure out how to define the Discord User Identity

Reply:
This is actually a work around to keep the compiler for yelling at us about it not being null when the object is constructed. We should probably make it nullable though instead.